tests/sensors/test_moisture.py

```diff
@@ -6,46 +6,58 @@
 DATA = {
     "config": {
         "battery": 100,
+        "offset": 0,
         "on": True,
         "reachable": True,
     },
-    "ep": 1,
-    "etag": "814610ebe8c84ea0c87e137ea0a3fee6",
-    "lastseen": "2021-08-15T06:58Z",
-    "manufacturername": "modkam.ru",
-    "modelid": "DIYRuZ_Flower",
-    "name": "Moisture 8",
+    "etag": "c35662de2333cf107ca22818d8d0a57b",
+    "lastannounced": "2023-05-13T10:25:54Z",
+    "lastseen": "2023-05-13T11:43Z",
+    "manufacturername": "_TZE200_myd45weu",
+    "modelid": "TS0601",
+    "name": "Soil Sensor",
     "state": {
-        "lastupdated": "2021-08-15T06:58:52.547",
-        "moisture": 69,
+        "lastupdated": "2023-05-13T11:43:05.976",
+        "lowbattery": False,
+        "moisture": 9000,
     },
-    "swversion": "22/07/2021 10:04",
+    "swversion": "1.0.8",
     "type": "ZHAMoisture",
     "uniqueid": "xx:xx:xx:xx:xx:xx:xx:xx-01-0408",
 }
 
 
+async def test_handler_moisture(mock_aioresponse, deconz_session, deconz_called_with):
+    """Verify that moisture sensor works."""
+    moisture = deconz_session.sensors.moisture
+
+    mock_aioresponse.put("http://host:80/api/apikey/sensors/0/config")
+    await moisture.set_config("0", offset=1)
+    assert deconz_called_with("put", path="/sensors/0/config", json={"offset": 1})
+
+
 async def test_sensor_moisture(deconz_sensor):
     """Verify that moisture sensor works."""
     sensor = await deconz_sensor(DATA)
 
-    assert sensor.moisture == 69
+    assert sensor.moisture == 9000
+    assert sensor.offset == 0
+    assert sensor.scaled_moisture == 90
 
     # DeconzSensor
     assert sensor.battery == 100
-    assert sensor.ep == 1
```

Comment:
Why are you removing ep?

Reply:
Only this comment left to clarify for this to be done 👍🏻

Reply:
I changed the test data and my apuli response has no ep attribute.

Reply:
I just wanted to know why it changed. No need to add it back